### DISCUSSION TITLE
OSPO Mind Map Final PR review

### DISCUSSION THREAD

**Discussion by anajsana** [todogroup/ospology#113] [Polls]
Choose the [Work Day Activity call](https://github.com/todogroup/work-day-activities) that works for most of the contributors (either the APAC & EMEA one or the AMER) in June to review comments and suggestions for the new addition to the [OSPO mind map](https://github.com/todogroup/ospology/tree/main/ospo-mindmap) project submitted by Thomas. Save this time to merge a [final version of the PR](https://github.com/todogroup/ospology/pull/111).

➡️ Please only participate if you are willing to **actively contribute to the review** 👍 Thanks!

🕐 You can use https://www.worldtimebuddy.com/ to check your timezone

**Comment by anajsana** [comment]
All votes went to:

* APAC & EMEA 📅 Monday, June 13 | 9:00am – 11:00am UTC

The topic will be discussed at June's APAC & EMER WD (next week)

**Reply by winterrocks** [reply]
I hope that I can make it, not 100% sure, but I will try to be there. If not, I did add several comments earlier.
